src/controller/python/templates/python-cluster-Objects-py.zapt

```diff
@@ -56,6 +56,17 @@ class {{asUpperCamelCase name}}(Cluster):
 
 {{/zcl_enums}}
 
+{{#zcl_bitmaps}}
+{{#first}}
+    class Enums:
```

Comment:
Is this really supposed to be Enums, not Bitmaps?

Reply:
thanks, updated